src/glib/mine/signalproc.h

```diff
@@ -107,6 +107,33 @@ class TSum {
 };
 
 /////////////////////////////////////////////////
+// Online Summa of sparse vector
+class TSumSpVec {
+private:
+	TVec<TIntFltKd> Sum; // current computed SUM value 
+	TUInt64 TmMSecs; // timestamp of current MA	    
+public:
+	TSumSpVec() {};
```

Comment:
needs some documentation on functionality and API (doxygen compatible using /// or ///<) for members and functions.